frontend/src/HomePage.css

```diff
@@ -0,0 +1,7 @@
+.homepage-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  min-height: 100vh; /*Specyifying 100 viewport height so that our footer can do margin-top: auto. This is a little trick so that the footer always fills up all remaing vertical space in flex col container and hence is always at bottom.*/
```

Comment:
If you move this property to the index.css on the root or main selector, is not better?